codegen/core/src/main/scala/com/lightbend/akkasls/codegen/SourceGeneratorUtils.scala

```diff
@@ -48,7 +50,16 @@ object SourceGeneratorUtils {
           .toList)
       .toSet
     if (packages.isEmpty) throw new IllegalArgumentException("Nothing to generate!")
-    longestCommonPrefix(packages.head, packages.tail)
+    longestCommonPrefix(packages.head, packages.tail) match {
+      case Nil =>
+        val packageNames = packages.map(_.mkString("."))
+        throw new RuntimeException(
+          s"No common package prefix found for protobuf packages [${packageNames.mkString(", ")}]. " +
+          "Auto-selection of root package for shared classes not possible, please configure explicit root package for " +
+          "Akka Serverless code generation in your build.") with NoStackTrace
```

Comment:
It would have been nice to include a link to documentation that describes how to define that in the build.

Reply:
Since this only happens with sbt, wdyt about just explicitly mentioning the sbt setting here?

I expect that it will be a corner case that most users won't end up with, so my thinking is that you only need to know about it if you need it - which will mean you end up with this error.

Reply:
Amended error message, let me know what you think, but already started release so not urgent, this will go in next release.

Reply:
lgtm, thanks!